app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+#
+class User < ActiveRecord::Base
+  has_secure_password
+
+  validates :email,
+            presence: true,
+            uniqueness: true,
```

Comment:
This is not case sensitive...

brian.e.mcelaney@gmail.com and Brian.E.McElaney@gmail.com

would be able to register for two seperate accounts